filter.imp

```diff
@@ -1,13 +1,111 @@
 [
-   { include: ["@.asm-generic/unaligned.h.", private, "<asm/unaligned.h>", "public"] },
+   # The following asm-generic files will always be replaced by asm files
```

Comment:
Careful, the use of the word `always` here is misleading.

More like:

If an architecture does not provide its own copy of arch/{$ARCH}/include/uapi/{foo}.h, then it will get a copy of include/uapi/asm-generic/{foo}.h which can still be #included via <asm/foo.h>.